portality/forms/application_processors.py

```diff
@@ -303,6 +303,10 @@ def finalise(self, account, save_target=True, email_alert=True):
             raise Exception("You cannot edit a not-existent application")
         if self.source.application_status == constants.APPLICATION_STATUS_ACCEPTED:
             raise Exception("You cannot edit applications which have been accepted into DOAJ.")
+        if self.form.application_status.data == "accepted" and (
```

Comment:
Please add test case for this

Reply:
Also, should use the value from `constants`, not the magic string